lib/Doctrine/ORM/EntityManager.php

```diff
@@ -222,12 +222,8 @@ public function getCache()
     /**
      * {@inheritDoc}
      */
-    public function transactional($func)
+    public function transactional(callable $func)
```

Comment:
Now (for 7.0+) `call_user_func($func, $this)` may also be replaced by `$func($this)` I think.